cmd/restic/cmd_backup.go

```diff
@@ -298,6 +299,11 @@ func collectRejectByNameFuncs(opts BackupOptions, repo *repository.Repository, t
 		if err != nil {
 			return nil, err
 		}
+
+		if valid, invalidPatterns := filter.ValidatePatterns(excludes); !valid {
```

Comment:
The first two errors seem to be a duplicate for the last two?

Reply:
I've tweaked the error messages to include the used flag, e.g. `--iexclude`